imports/api/EloCalculator.js

```diff
@@ -1,8 +1,8 @@
-import { Players } from './Players.js';
+import Players from './Players';
+import Constants from './Constants';
+import GameTypeUtils from './utils/GameTypeUtils';
 
-import { Constants } from './Constants.js';
-
-export class EloCalculator {
+export default class EloCalculator {
```

Comment:
We want to indicate that we are in fact exporting this class, because we want to be able to call the constructor on this object. I don't think you're able to declare a class without a name, so we'll have to deal with this for now.